src/transaction/utils.ts

```diff
@@ -56,6 +57,37 @@ function scaleValue(value, multiplier, extra = 0) {
   return (new BigNumber(value)).times(multiplier).plus(extra).toString()
 }
 
+/**
+ * @typedef {Object} ClassicAccountAndTag
+ * @property {string} classicAccount - The classic account address.
+ * @property {number | false | undefined } tag - The destination tag;
+ *                    `false` if no tag should be used;
+ *                    `undefined` if the input could not specify whether a tag should be used.
+ */
+
+/**
+ * Given an address (account), get the classic account and tag.
+ *
+ * @param Account The address to parse.
+ * @returns {ClassicAccountAndTag}
+ *          The classic account and tag.
+ */
+function getClassicAccountAndTag(Account: string): {classicAccount: string, tag: number | false | undefined} {
+  try {
```

Comment:
Don't use try / catch to do flow control.

Reply:
What alternative would you suggest? In this case, keep in mind that we want to decode the X-address only once.

Reply:
This is a strong anti pattern and there should be compelling evidence that this deviation is warranted by things that are out of our control. 

```javascript 
if (isXAddress) {
  // handle x address
} else {
  // handle classic address
}
```

I don't think that decoding is a big deal for the code clarity it buys us.  If this really bugs you, then you can replace the top `if` with 'starts with X || starts with T'.